src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1310,10 +1311,10 @@ public static WorldGenSettings loadDimensionsWithServerSeed(WorldGenSettings wgs
                 seededRegistry.register(key, dimension, originalRegistry.lifecycle(dimension));
             }
         }
-        
+
         return new WorldGenSettings(seed, generateFeatures, generateBonusChest, seededRegistry, legacyCustomOptions);
     }
-    
+
```

Comment:
I don't understand why you're changing so much whitespace.

Reply:
I don't know either